src/util.js

```diff
@@ -32,16 +32,22 @@ function loadYamlFile(filePath) {
 }
 
 
-function loadFile(filePath) {
-  let fileData;
-  if (fs.existsSync(`${filePath}.js`)) {
-    fileData = require(`${filePath}.js`) // eslint-disable-line import/no-dynamic-require, global-require
-  } else if (fs.existsSync(`${filePath}.yaml`)) {
-    fileData = loadYamlFile(`${filePath}.yaml`)
-  } else if (fs.existsSync(filePath)) {
-    fileData = loadYamlFile(filePath)
+function loadFile(fileToLoad) {
+  const candidateFileNames = [fileToLoad, `${fileToLoad}.js`, `${fileToLoad}.yaml`, `${fileToLoad}.yml`];
```

Comment:
@dreich-nucleus Checkout the change I proposed here, I used `find` so we do not test file if any of then return true the lop stops. We have created this array, so we remove the code duplication from the old implementation. And Added a clear error if the all process fails, the user will have a good idea what did we look for and why it failed.